traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/PartnerOffering.java

```diff
@@ -51,47 +50,29 @@ public int getAvailableQuantity() {
         return availableQuantity;
     }
 
+    public PurchasePass toPurchasePass() {
```

Comment:
Add a comment here explaining that this is a workaround regarding the handling of purchases in the purchase form, that is technical debt and that in the future it should be refactored.